pkg/pluginmanager/manager.go

```diff
@@ -209,7 +209,7 @@ func getAdditionalTestPluginDiscoveries() []configtypes.PluginDiscovery {
 	for idx, image := range testDiscoveryImages {
 		testDiscoveries = append(testDiscoveries, configtypes.PluginDiscovery{
 			OCI: &configtypes.OCIDiscovery{
-				Name:  fmt.Sprintf("test_%d", idx),
```

Comment:
self: Modified the OCI discovery source name  from `test_` to `disc_` to be generic. Would like to hear opinions on this.